.pre-commit-config.yaml

```
@@ -0,0 +1,16 @@
+# Configuration file for pre-commit (https://pre-commit.com/).
+# Please run `pre-commit run --all-files` when adding or changing entries.
+
+repos:
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.8.3
```

```suggestion
    rev: v0.9.9
```

Same version in pyproject.toml